UPGRADE-1.9.md

```diff
@@ -76,6 +48,52 @@
     +           permanent: false
     ```
 
+#### Upgrades Symfony to v5.2
+
+1. Upgrade Symfony dependencies by:
+
+    ```bash
+    composer config extra.symfony.require "^5.2"
+    composer require "symfony/browser-kit" "^5.2"
+    composer require "symfony/debug-bundle" "^5.2"
+    composer require "symfony/intl" "^5.2"
+    composer require "symfony/web-profiler-bundle" "^5.2"
```

Comment:
```suggestion
    composer require --dev "symfony/browser-kit" "^5.2" --no-update --no-scripts
    composer require --dev "symfony/debug-bundle" "^5.2" --no-update --no-scripts
    composer require --dev "symfony/intl" "^5.2" --no-update --no-scripts
    composer require --dev "symfony/web-profiler-bundle" "^5.2" --no-update --no-scripts
    composer update
```

Reply:
With the current commands, it would resolve dependencies 4 times (and add them to regular require).